modules/src/ics03_connection/msgs.rs

```diff
@@ -139,46 +143,52 @@ impl Msg for MsgConnectionOpenInit {
 pub struct MsgConnectionOpenTry {
     connection_id: ConnectionId,
     client_id: ClientId,
+    client_state: Option<AnyClientState>,
     counterparty: Counterparty,
     counterparty_versions: Vec<String>,
     proofs: Proofs,
     signer: AccountId,
 }
 
 impl MsgConnectionOpenTry {
-    /// Getter for accessing the `consensus_height` field from this message. Returns the special
-    /// value `0` if this field is not set.
-    pub fn consensus_height(&self) -> Height {
-        match self.proofs.consensus_proof() {
-            None => Height(0),
-            Some(p) => p.height(),
-        }
-    }
-
-    /// Getter for accesing the whole counterparty of this message. Returns a `clone()`.
-    pub fn counterparty(&self) -> Counterparty {
-        self.counterparty.clone()
+    /// Getter for accessing the connection identifier of this message.
```

Comment:
Did you reorder these functions? I'm just trying to understand why is this diff so complex, because nothing seems to have changed except for the order of functions (and there is `client_state` getter new).

Reply:
Yes, I ordered the getters consistent with the order in the struct.

Reply:
Good idea, I like this. I'll try to do the same in the future.